SeaPkg/Tools/GenSeaArtifacts/gen_aux/readme.md

```diff
@@ -46,14 +46,17 @@ offset = 'Optional[u32]'
 
 ### rule

```

Comment:
Leaving a comment as a note to hold off on completing this PR.  I want to get my PR(s) in before this so that additional change isn't required (for now) for my config file PR.